packages/client/src/checkout/types/checkoutOrderOperation.types.ts

```diff
@@ -0,0 +1,9 @@
+import type { CheckoutOrderOperationChange } from './checkoutOrderOperationChange.types';
+import type { CheckoutViolation } from './checkoutViolation.types';
+
+export type CheckoutOrderOperation = {
```

Comment:
Can you please confirm if all these properties are indeed optional? Seems weird for an endpoint response 😕

Reply:
CheckoutOrderOperation was created based on [this](https://ffdevelopers.farfetch.tech/api/commerce/def/checkoutorderoperation.html)

Reply:
Got it, better follow the docs! Stills sus tho 🤨